package.json

```diff
@@ -15,7 +15,13 @@
   "engines": {
     "node": ">=4"
   },
+  "size-limit": [
+    {
+      "path": "index.js"
```

Comment:
I'd set an explicit `limit` here, so that people don't search for `size-limit` docs and look what is the default size limit.

Reply:
I think it's ok not to do this.

Reply:
Why? `size-limit` isn't popular enough so that everyone knows what the defaults are.

Reply:
Hmm... my understanding is that there isn't a default size limit and it must be set explicitly (let me know if that's not the case), so no limit is enforced and most folks don't need to know what this tool does or what the defaults are.

As I mentioned in my comments, this isn't being used as a tool to enforce the size limit.